packages/connect/src/http_client.ts

```diff
@@ -115,10 +118,13 @@ export class HttpClient implements Client {
         const requestOpts = {
             payload: request,
         };
-        const fees = await this._requestAsync('/fees', HttpRequestType.Post, requestOpts);
-        assert.doesConformToSchema('fees', fees, schemas.relayerApiFeesResponseSchema);
-        typeConverters.convertStringsFieldsToBigNumbers(fees, ['makerFee', 'takerFee']);
-        return fees;
+        const result = await this._requestAsync(
+            '/fees',
+            HttpRequestType.Post,
+            relayerResponseJsonParsers.parseFeesResponseJson,
```

Comment:
Why rather pass these methods in then simply calling them on the return value of `_requestAsync`. I would prefer the latter. It's clearer to me.